version.txt

```diff
@@ -1 +1 @@
-1.0.1
+1.0.1+build.2
```

Comment:
Is there a reason you're bumping to `build.2`? Just for context I am of the thinking that `1.0.1` is implicitly `1.0.1+build.0`, so the next build after that would be `build.1`.

Reply:
The code hasn't changed, so 1.0.1+build.1 is the version for the pre-release (staging AMI) that I test in COOL staging. Then I have to change the version again so I can create a release (production AMI).

Reply:
Hm, alright. In cases like this would it make more sense to edit the release and change it from a pre-release to a regular release instead? That _should_ trigger the correct workflow as the documentation for the `released` event states:
> A release was published, or a pre-release was changed to a release.

Reply:
Next time I could bump the version to the next RC (1.0.1-rc.2 in this example), build the pre-release, then finalize and bump the build version for the release (1.0.1+build.1 in this example). Would that work for you? 

I like having both the release and pre-release around for historical reasons as sometimes a system package updates and breaks something between when I build the pre-release and when I build the release.

Reply:
If you prefer to have both the pre-release and release on GitHub around then it's fine. It doesn't hurt anything to keep incrementing the build number. I just thought it might be more straightforward the other way since the staging AMI would persist.